src/start/CompatibilityScreen.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+import { Logo } from '../common';
+import { BRAND_COLOR } from '../common/styles';
+
+const styles = StyleSheet.create({
+  screen: {
+    flex: 1,
+    justifyContent: 'center',
+    backgroundColor: BRAND_COLOR,
+  },
+  text: {
+    textAlign: 'center',
+    color: 'white',
+    fontSize: 20,
+    margin: 10,
+  }
+});
+
+export default class CompatibilityScreen extends React.Component {
+
+  render() {
+    return (
+      <View style={styles.screen}>
+        <Logo />
+        <Text style={styles.text}>This app is too old!</Text>
```

Comment:
I have added a task or later. When we publish both the iOS and Android apps. We don't have the links yet, so leaving as is for now.